dynawo/sources/Models/Modelica/PreassembledModels/GeneratorSynchronousFourWindingsPmConstAc8b.xml

```diff
@@ -15,16 +15,12 @@
   <!-- Machine with constant mechanical power and a voltage regulator (Ac8b). -->
   <dyn:modelicaModel id="GeneratorSynchronousFourWindingsPmConstAc8b">
     <dyn:unitDynamicModel id="generator" name="Dynawo.Electrical.Machines.OmegaRef.GeneratorSynchronous" initName="Dynawo.Electrical.Machines.OmegaRef.GeneratorSynchronousExt4W_INIT"/>
-    <dyn:unitDynamicModel id="voltageRegulator" name="Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.Ac8b" initName="Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.Ac8b_INIT"/>
+    <dyn:unitDynamicModel id="voltageRegulator" name="Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.Ac8b" initName="Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.Ac16c_INIT"/>
```

Comment:
Maybe we could rename the init Ac16_INIT so we know it is also used for Ac8b and Ac1a, Ac6a Ac1a. Also, we should add these infos to the description.
I could be named AcBase_INIT.mo, for example.

Reply:
Ac168_INIT.mo is a good and accurate name.

The description has been completed.

Reply:
It is not an init model for Ac1a, I think these names can be confusing.
But OK for now, it will added in the description anyway for which model it corresponds.